src/modules/sections/projects-section.tsx

```
@@ -1,46 +1,35 @@
-        rel="noopener noreferrer"
-        class="theme-link with-arrow"
-      >
-        GitHub
-      </a>
-      .
-    </Motion.p>
-  );
-}
+  export function ProjectsSection() {
+    return (
+      <div> // this will be a  `block` element in the CMS which is required if you want to add any widget (e.g. text) to the page.
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)